experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -45,6 +45,26 @@ def get_form_kwargs(self):
         return kwargs
 
 
+class ValidationErrorsMixin:
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        show_errors = self.request.GET.get("show_errors", "") == "true"
+        errors = self.get_object().get_invalid_fields_errors()
+
+        validation_errors = {}
+
+        if show_errors:
+            for field, error in errors:
+                if field not in validation_errors:
+                    validation_errors[field] = []
+                validation_errors[field].append(error)
```

Comment:
Looks like we start in the `{field: [errors}}` structure coming out of the serializer, then convert to `[(field, error)]` coming out of `get_invalid_fields_errors`, I guess to make the page lookup a little easier, and then convert BACK to `{field: [errors]}` to actually show in the template.  

Seems more like we should just return the original serializer structure out of `get_invalid_fields_errors`, look up the fields with errors to map to pages from that, and then use that structure directly to pass into the template since that's already the structure we'd want?